lib/bios/seabios.robot

```diff
@@ -0,0 +1,305 @@
+    # Lines to strip:
+    ${out}=    Read From Terminal Until    ${checkpoint}
+    ${menu}=    Parse Menu Snapshot Into Construction    ${out}    7    0
+    RETURN    ${menu}
+
+Parse Menu Snapshot Into Construction
+    [Documentation]    Breaks grabbed menu data into lines.
+    [Arguments]    ${menu}    ${lines_top}    ${lines_bot}
+    ${slice_start}=    Set Variable    ${lines_top}
+    IF    ${lines_bot} == 0
+        ${slice_end}=    Set Variable    None
+    ELSE
+        ${slice_end}=    Evaluate    ${lines_bot} * -1
+    END
+    ${menu}=    Remove String    ${menu}    \r
+    @{menu_lines}=    Split To Lines    ${menu}
+    @{construction}=    Create List
+    FOR    ${line}    IN    @{menu_lines}
+        # Replace multiple spaces with a single one
+        ${line}=    Replace String Using Regexp    ${line}    ${SPACE}+    ${SPACE}
+        # Remove leading and trailing spaces
+        ${line}=    Strip String    ${line}
+        # Drop leading and trailing pipes (e.g. in One Time Boot Menu)
+        ${line}=    Strip String    ${line}    characters=|
+        # Remove leading and trailing spaces
+        ${line}=    Strip String    ${line}
+        # Drop all remaining borders
+        ${line}=    Remove String Using Regexp    ${line}    ^[\\|\\s/\\\\-]+$
+        # If the resulting line is not empty, add it as a menu entry
+        ${length}=    Get Length    ${line}
+        IF    ${length} > 0    Append To List    ${construction}    ${line}
+    END
+    Log    ${construction}
+    ${construction}=    Get Slice From List    ${construction}    ${slice_start}    ${slice_end}
+    # TODO: Improve parsing of the menu into construction. It can probably be
+    # simplified, but at least we have this only in one kewyrod not in multiple
+    # ones.
+    # Make sure to remove control help text appearing in the screen if somehow
+    # they are still there.
+    Remove Values From List
+    ...    ${construction}
+    ...    Esc\=Exit
+    ...    ^v\=Move High
+    ...    <Enter>\=Select Entry
+    ...    F9\=Reset to Defaults F10\=Save
+    ...    LCtrl+LAlt+F12\=Save screenshot
+    ...    <Spacebar>Toggle Checkbox
+    ...    one adjusts to change
+    ...    Select boot device:
+    ...    , N for PXE boot
+    RETURN    ${construction}
+
+Enter Setup Menu SeaBIOS And Return Construction
+    [Documentation]    Enters Setup Menu and returns Setup Menu construction
+    Enter Setup Menu SeaBIOS
+    ${menu}=    Get Setup Menu Construction
+    RETURN    ${menu}
+
+Get RTC Clock Submenu Construction
+    [Arguments]    ${checkpoint}=Esc=Exit    ${lines_top}=1    ${lines_bot}=1    ${opt_only}="${FALSE}"
+    # In most cases, we need to strip two lines:
+    #    TOP:
+    #    Title line, such as:    Dasharo System Features
+    #    BOTTOM:
+    #    Help line, such as:    F9=Reset to Defaults    Esc=Exit
+    ${submenu}=    Get Menu Construction    ${checkpoint}    ${lines_top}    ${lines_bot}
```

Comment:
This keyword is not defined aywhere? Since `Get RTC Clock Submenu Construction` uses unused keyword, I assume it has not been tested. 

Which keyword from this file do you actually need to execute the tests, and which have been tested @pietrushnic ?